src/Models/ApiClientScope.php

```diff
@@ -7,4 +7,19 @@
 class ApiClientScope extends Model
 {
     protected $fillable = ['name','description'];
+
+    public function getRouteKeyName()
+    {
+        return 'name';
+    }
+
+    public function scopeFindByNames($query, array $names = [])
+    {
+        return $query->whereIn('name', $names)->get();
+    }
+
+    public function scopeName($query, string $name)
```

Comment:
This scope is useless. We can use `ApiClientScope::whereName('test')` instead without defining new functions.

Reply:
I prefer not to use magic that much, but ok